content/docs/about/kubeflow.md

```diff
@@ -11,6 +11,44 @@ recreate other services, but to provide a straightforward way to deploy
 best-of-breed open-source systems for ML to diverse infrastructures. Anywhere 
 you are running Kubernetes, you should be able to run Kubeflow.
 
+## Getting started with Kubeflow
+
+Follow the [getting-started guide](/docs/started/getting-started) to set up your
+environment.
+
+Then read the [documentation](/docs/) to learn how to use Kubelow.
+Highlights include the following:
```

Comment:
```suggestion

```

Reply:
Remove the word "highlights", which might not be as familiar for ESL readers. Also, eliminate the sentence fragment by specifying "following documents" or using a similar noun.

Reply:
These suggested changes are very hard to see in the GitHub UI. I don't want to just commit the change without seeing what it is, and I'm doing my edits locally and want to avoid sync/merge conflicts. I'll update the sentence to the what I think you mean. :)